src/api/todos.ts

```diff
@@ -0,0 +1,34 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = async (userId: number) => {
+  const todos = await client.get<Todo[]>(`/todos?userId=${userId}`);
+
+  return todos;
+};
+
+export const addTodo = async (todo: Omit<Todo, 'id'>) => {
+  const add = await client.post<Todo>('/todos', todo);
+
+  return add;
+};
+
+export const deleteTodo = async (todoID: string) => {
+  const deletedTodo = await client.delete(todoID);
```

Comment:
why can't we pass endpoint only here instead of passing everywhere where we use this function?